spec/features/support/sign_up_helpers.rb

```diff
@@ -0,0 +1,23 @@
+def sign_up_for_account(email: 'default@gov.uk')
+  visit new_user_registration_path
+  fill_in 'user_email', with: email
+  click_on 'Sign up'
+end
+
+def create_password_for_account(password: 'supersecret')
+  return unless confirmation_email_received?
```

Comment:
Did this to make acceptance tests nicer - can share the simple sign up / create password `before` block - but not sure if it's worth it.

Is this too hidden?